source/code-snippets/authentication/aws.js

```diff
@@ -10,10 +10,6 @@ const authMechanism = "MONGODB-AWS";
 let uri =
   `mongodb+srv://${accessKeyId}:${secretAccessKey}@${clusterUrl}/?authMechanism=${authMechanism}`;
```

Comment:
This currently fails as it seems the Node driver still requires the `authSource` be specified.